src/Microsoft.DotNet.XHarness.CLI/Commands/Android/AndroidRunCommand.cs

```diff
@@ -104,6 +101,9 @@ protected override Task<ExitCode> InvokeInternal(ILogger logger)
 
             int instrumentationExitCode = (int)ExitCode.GENERAL_FAILURE;
 
+            // Empty log as we'll be uploading the full logcat for this execution
```

Comment:
This `InvokeHelper` is called from within different commands and that's why it was moved?

Reply:
yes, from `test` as well